auth/user_mgt.go

```diff
@@ -170,14 +174,16 @@ func convertMultiFactorInfoToServerFormat(mfaInfo MultiFactorInfo) (multiFactorI
 	if mfaInfo.EnrollmentTimestamp != 0 {
 		authFactorInfo.EnrolledAt = time.Unix(mfaInfo.EnrollmentTimestamp, 0).Format("2006-01-02T15:04:05Z07:00Z")
 	}
+	if mfaInfo.UID != "" {
```

Comment:
Since phone is the only factorId, is it ok to put this in the if block below?

Or is this to cover the case where a factorId is not specified?

Reply:
It can be moved to the block bellow of course, but why we have the `EnrollmentTimestamp` at the top for example?

I just thought that `UID` will probably exist for every "factor", because we will need it when updating something, so it's safe to move it from the block

Reply:
Good point. We will likely move it out of the if block when we add another MFA (like TOTP).

Similarly, DisplayName is a generic field too. So you could move that out too.